save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/table/filters/VulnerabilitiesFiltersRow.kt

```diff
@@ -129,9 +129,10 @@ val vulnerabilitiesFiltersRow: FC<VulnerabilitiesFiltersProps> = FC { props ->
                 buttonBuilder(faSearch, classes = "btn mr-1", isOutline = props.filter == filter, style = "secondary") {
                     props.onChangeFilter(filter)
                 }
-                buttonBuilder(faWindowClose, classes = "btn mr-1", isOutline = true, style = "secondary") {
+                buttonBuilder(faWindowClose, classes = "btn mr-1", title = "Drop filters", isOutline = true, style = "secondary") {
                     props.onChangeFilter(null)
-                    setFilter { props.filter }
+                    // need to drop all tags
+                    setFilter { props.filter.copy(tags = emptySet()) }
```

Comment:
@sanyavertolet I decided to drop all tags when we drop filters, previously it was not very obvious. Drop means that we need to drop all filters, including tags

Reply:
oh ok